### PR TITLE
Add element picker to dev window

### PR DESCRIPTION
**Pull request by DrSmugleaf**
https://user-images.githubusercontent.com/10968691/186221670-8a116ddd-0524-42a4-a092-81ab4b6a295e.mp4

technology

### CODE REVIEW

**Comment by Visne** [comment]
Wtf how

**Review by metalgearsloth — APPROVED** [review]

**Comment by metalgearsloth** [comment]
we have the technology

File: Robust.Client/UserInterface/DevWindow/DevWindowTabUI.xaml.cs [hunk 1]
```diff
@@ -25,12 +32,70 @@ public DevWindowTabUI()
         private void InitializeComponent()
         {
             RobustXamlLoader.Load(this);
+            IoCManager.InjectDependencies(this);
 
-            ControlTreeRoot.OnKeyBindDown += ControlTreeRootOnOnKeyBindDown;
+            _clyde.MouseMove += OnMouseMove;
+            _input.UIKeyBindStateChanged += OnUIKeyBindStateChanged;
```
Comment (PJB3005) [inline Robust.Client/UserInterface/DevWindow/DevWindowTabUI.xaml.cs]: You're not removing these subscriptions anywhere. Add them in `EnteredTree` and remove them in `ExitedTree`.

File: Robust.Client/UserInterface/DevWindow/DevWindowTabUI.xaml.cs [hunk 2]
```diff
@@ -25,12 +32,70 @@ public DevWindowTabUI()
...
+
+            foreach (var subEntry in entryStack)
+            {
+                ControlMap[subEntry].Open();
+            }
+
+            SelectControl(controlUnderMouse);
+        }
+
+        private bool OnUIKeyBindStateChanged(BoundKeyEventArgs arg)
```
Comment (PJB3005) [inline Robust.Client/UserInterface/DevWindow/DevWindowTabUI.xaml.cs:83]: Do you not want to check which UI key fired? Limit it to click?

**Review by PJB3005 — CHANGES_REQUESTED** [review]

File: Robust.Client/UserInterface/DevWindow/DevWindowTabUI.xaml.cs (hunk 1 above)
Comment (DrSmugleaf) [reply Robust.Client/UserInterface/DevWindow/DevWindowTabUI.xaml.cs]: Fixed in [`5a35f94` (#3180)](https://github.com/space-wizards/RobustToolbox/pull/3180/commits/5a35f9446949ea6dc693d8e48a7220fbfe6d3d07)

File: Robust.Client/UserInterface/DevWindow/DevWindowTabUI.xaml.cs (hunk 2 above)
Comment (DrSmugleaf) [reply Robust.Client/UserInterface/DevWindow/DevWindowTabUI.xaml.cs:83]: Fixed in [`5a35f94` (#3180)](https://github.com/space-wizards/RobustToolbox/pull/3180/commits/5a35f9446949ea6dc693d8e48a7220fbfe6d3d07)

Comment (DrSmugleaf) [reply Robust.Client/UserInterface/DevWindow/DevWindowTabUI.xaml.cs:83]: Fixed in [`5a35f94` (#3180)](https://github.com/space-wizards/RobustToolbox/pull/3180/commits/5a35f9446949ea6dc693d8e48a7220fbfe6d3d07)

**Pull request merged**